.github/workflows/build.yml

```diff
@@ -227,7 +227,102 @@ jobs:
+                -Aclippy::len_zero \
+                -Aclippy::let_and_return \
+                -Aclippy::manual_filter \
+                -Aclippy::manual_map \
+                -Aclippy::manual_memcpy \
+                -Aclippy::manual_range_contains \
+                -Aclippy::manual_range_patterns \
+                -Aclippy::manual_saturating_arithmetic \
+                -Aclippy::manual_strip \
+                -Aclippy::map_clone \
+                -Aclippy::map_flatten \
+                -Aclippy::match_like_matches_macro \
+                -Aclippy::match_ref_pats \
+                -Aclippy::mut_mutex_lock \
+                -Aclippy::needless_bool \
+                -Aclippy::needless_borrow \
+                -Aclippy::needless_borrowed_reference \
+                -Aclippy::needless_borrows_for_generic_args \
+                -Aclippy::needless_lifetimes \
+                -Aclippy::needless_question_mark \
+                -Aclippy::needless_range_loop \
+                -Aclippy::needless_return \
+                -Aclippy::new_without_default \
+                -Aclippy::non_minimal_cfg \
+                -Aclippy::option_as_ref_deref \
+                -Aclippy::option_map_or_none \
+                -Aclippy::option_map_unit_fn \
+                -Aclippy::precedence \
+                -Aclippy::ptr_arg \
+                -Aclippy::question_mark \
+                -Aclippy::readonly_write_lock \
+                -Aclippy::redundant_closure \
+                -Aclippy::redundant_field_names \
+                -Aclippy::redundant_guards \
+                -Aclippy::redundant_pattern_matching \
+                -Aclippy::redundant_slicing \
+                -Aclippy::redundant_static_lifetimes \
+                -Aclippy::result_large_err \
+                -Aclippy::result_unit_err \
+                -Aclippy::search_is_some \
+                -Aclippy::single_char_pattern \
+                -Aclippy::single_match \
+                -Aclippy::slow_vector_initialization \
+                -Aclippy::tabs_in_doc_comments \
+                -Aclippy::to_string_in_format_args \
+                -Aclippy::too_many_arguments \
+                -Aclippy::toplevel_ref_arg \
+                -Aclippy::type_complexity \
+                -Aclippy::unnecessary_cast \
+                -Aclippy::unnecessary_get_then_check \
+                -Aclippy::unnecessary_lazy_evaluations \
+                -Aclippy::unnecessary_mut_passed \
+                -Aclippy::unnecessary_sort_by \
+                -Aclippy::unnecessary_to_owned \
+                -Aclippy::unnecessary_unwrap \
+                -Aclippy::unused_unit \
+                -Aclippy::unwrap_or_default \
+                -Aclippy::useless_conversion \
+                -Aclippy::useless_format \
+                -Aclippy::vec_init_then_push \
+                -Aclippy::while_let_on_iterator \
+                -Aclippy::wrong_self_convention \
+                -Dclippy::try_err
```

Comment:
If we deny warnings by default, we can remove this deny exception.

Reply:
Done, I removed this deny exception.